src/aui-toolbar/js/aui-toolbar.js

```diff
@@ -389,7 +392,12 @@ ToolbarRenderer.prototype = {
             }
 
             // Add cssClass support
-            cssClass = [CSS_BTN, CSS_BTN_DEFAULT, value.cssClass];
+            cssClass = [CSS_BTN_TOOLBAR, value.cssClass];
+
+            if (value.addDefaultButtonCssClasses !== false) {
```

Comment:
Since this code was added here we need to add test cases for it on aui-toolbar.js.